app/uk/gov/hmrc/customs/notification/controllers/CustomsNotificationBlockedController.scala

```diff
@@ -38,7 +38,7 @@ class CustomsNotificationBlockedController @Inject()(val logger: NotificationLog
   def blockedCount(): Action[AnyContent] = Action.async {
     implicit request =>
```

Comment:
I think its worth extracting the header validation code to a private function as it is the same. An differences could be parameterised - eg endpoint name

Reply:
Not entirely convinced but moved to function. Open to suggestions on simplifying it.